Adding 8th and 9th PCs into PC Mode

Hillosanation opened the pull request:
L2790-2808 was a nitpick, can revert if need be.
Changed storage of PC number from leftover pieces to specific PC numbers
used different algo to shuffle bag.

Hillosanation committed:
Added 8th, 9th PC functionality

i dont know how to log changes

-implemented 8th and 9th bags
-removed period for bags using "th"
-store PC number instead of leftover pieces

Hillosanation committed:
remove test

Hillosanation committed:
update perfect-clear mode comment

Hillosanation committed:
Delete Tetris.exe

torchlight commented:
I'd rather just add support for dupe PCs in general, rather than have 8th and 9th as special cases. The thing about 9th is that it's not that different from other non-9th dupe 2nds: most cases can be boxed (all except TTJZ/TTLS), and TTJZ/TTLS can be handled with the same setup as TTSZ (3p PCO hold T).

Hillosanation commented:
yeah that's a good idea

Hillosanation committed:
Support all dupe bags in PC mode

9th now is a subset of possible dupe 2nds given
The comment is too long in L1200, but idk how to change the height of the comment

Hillosanation committed:
oops
